docs/bugzilla.md

```diff
@@ -107,3 +108,16 @@ users.
 ```python
 t.add_cc(['username1@mail.com', 'username2@mail.com'])
 ```
+
+### add_attachment(self, data, file_name, summary, \*\*kwargs ) <a name="add_attachment"></a>
```

Comment:
Can we put the named parameters in the following order:

file_name, data, summary